doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -75,6 +75,9 @@ nRF Desktop
 * Added:
 
   * Added possibility to ask for bootloader variant using config channel.
+  * Added working with ChromeBook (WWCB) configuration.
```

Comment:
@greg-fer, should we add the wwcb configuration to the configuration list in README ?

Reply:
Yes, we should add it.